sdk/messaging/azwebpubsub/internal/util.go

```diff
@@ -0,0 +1,80 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package internal
+
+import (
+	"errors"
+	"net/url"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+)
+
+const (
+	TokenScope = "https://webpubsub.azure.com/.default"
+)
+
+func NewPipeline(plOpts runtime.PipelineOptions, options *policy.ClientOptions) runtime.Pipeline {
```

Comment:
I think this can be removed as you're using `azcore.NewClient()` which creates a pipeline?

Reply:
move to test where it is used